app/views/admin/content/_form.html.erb

```diff
@@ -259,3 +262,22 @@
     </div>
   </div>
 </div>
+
+<div class="modal fade" id="answerReveal" tabindex="-1" role="dialog" aria-labelledby="answerReveal" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h2>Answer/Reveal snippet</h2>
+      </div>
+      <div class="modal-body">
+        <p>Copy the snippet below and paste it to the desired place by clicking on the 'Source' button. <b>Note:</b> Add a \n to the answer to start a new paragraph.</p>
+        <pre id="ma_says--snippet-container">
+&lt;hr class=&quot;snippet&quot; data-snippet-name=&quot;answerreveal&quot; data-snippet-text-question=&quot;<span style="background:red;color:#fff;">[ENTER QUESTION TEXT HERE]</span>&quot; data-snippet-text-answer=&quot;<span style="background:red;color:#fff;">[ENTER ANSWER TEXT HERE]</span>&quot; data-snippet-text-callout=&quot;<span style="background:red;color:#fff;">[ENTER CALLOUT TEXT HERE]</span>&quot; data-snippet-url-callout=&quot;<span style="background:red;color:#fff;">[ENTER CALLOUT URL HERE]</span>&quot; /&gt;
```

Comment:
xmp seems to be only one which doesn't need escaping however its a deprecated tag.

The suggestion of a textarea which is disabled seems like a good idea, if it means we get rid of horrible escaped tags in the markup.